app/views/core_induction_programme/lesson_parts/show.html.erb

```diff
@@ -54,7 +54,7 @@
       <% end %>
     <% end %>
 
-    <nav class="step-by-step-navigation" role="navigation">
+    <nav class="step-by-step-navigation" role="navigation" aria-label="lesson-part-navigation">
```

Comment:
What's with the hyphens?

Reply:
Lack of thought yesterday evening is what it is :P